### ISSUE TITLE
bbpress2 imported users' passwords are not working

### ISSUE THREAD

**Issue by bulentsoylu** [vanilla/porter#217]
Hi dear developers,
I couldn't solve this problem in community forum and bring it here.
I've successfully imported users from my client's previous forum with hash method Vanilla.
Although SSO is successful:
![image](https://user-images.githubusercontent.com/26273592/146565029-7ee9f897-aa73-4fe9-885c-5171d1e2fe1b.png)
I've opted approval method for registration. Because I couldn't manage to have it worked SSO.
Forum offers two options to login. Both are not letting my user logged in after import.
![image](https://user-images.githubusercontent.com/26273592/146557157-824c64cf-a359-4a58-b00b-1383ca419362.png)
when I try to login with a user I've created in source wp site I can't.
![image](https://user-images.githubusercontent.com/26273592/146556909-e3b47704-ee7c-4e2a-86f0-93b8481df26d.png)

will you please help me get these users login with their password in source wp site?

We don't want them to register again or go to process of forget password. when you use that forget method it's logging in even you enter a different password if you chose to login via wp site login option it logs in.

Note: I'm using Vanilla Forum plugin in source web  site Version: 1.3.1

thanks

**Comment by bulentsoylu** [comment]
I've removed the connection to the wp and imported again. As a standalone Forum the passwords aren't working.

**Comment by bulentsoylu** [comment]
Guys, It turned out like this:
in vanilla2export.php--> bbPress 2 section  wp_usermeta table is used to fetch the membership status of users . it was looking wp_capabilities meta_key which was different in my client's site. it's hosted in Godaddy and the wp_  another suffix before it  was like wp_7080980_capabilities. So it couldn't get  status of users from this meta_key. First I replaced that meta_key with  wp_98080_capabilities value and it was assigning all the rest 1, Guest status I've changed it to 4 member status. after that,  logins to the forum using SSO worked.  The problem was not password reading, it was related to assigning user status while exporting/importing via Vanilla2Export.php.  

![image](https://user-images.githubusercontent.com/26273592/146698480-940bd76a-60dc-4881-98a4-dda9dd452cbd.png)

This issue is solved but embedding issue is still there. I will open another thread for it. 
thanks
